testing/helpers/wrappers/tabsWrappers.js

```diff
@@ -0,0 +1,78 @@
+import $ from 'jquery';
+import { extend } from 'core/utils/extend';
+
+export class TestTabsWrapper {
+    constructor($element, options) {
+        const { itemsCount } = options;
+        const tabsOptions = extend({
+            items: this._generateItems(itemsCount),
+        }, options);
+
+        this._$tabs = $($element).dxTabs(tabsOptions);
+        this._tabs = this._$tabs.dxTabs('instance');
+    }
+
+    _generateItems(count = 5) {
+        return [...new Array(count)].map((value, index) => ({ text: `item ${index}` }));
+    }
+
+    _checkTabsContent($tabs) {
+        const items = this._tabs.option('items');
+        $tabs.toArray().forEach((tabElement, index) =>
+            QUnit.assert.equal($(tabElement).text(), items[index].text, `text of the ${index} tab`)
+        );
+    }
+
+    set width(value) {
+        this._tabs.option('width', value);
+    }
+
+    setItemsByCount(count) {
+        this._tabs.option('items', this._generateItems(count));
+    }
+
+    checkTabsWithoutScrollable() {
+        const $tabs = this._$tabs.find('> .dx-tabs-wrapper > .dx-tab');
```

Comment:
I would prefer this._$tabs.find('.dx-tab') instead of '> .dx-tabs-wrapper > .dx-tab' (full path) if it is possible.
And, the same in the next function: this._$tabs.find('> .dx-tabs-scrollable .dx-scrollable-content > .dx-tabs-wrapper > .dx-tab')